pkg/piecestore/psclient/readerwriter.go

```diff
@@ -71,19 +71,23 @@ func (s *StreamWriter) Close() error {
 		return err
 	}
 
-	if err := auth.VerifyMsg(reply.SignedHash, s.signer.remoteID); err != nil {
+	s.storagenodeHash = reply.SignedHash
+
+	zap.S().Debugf("Stream close and recv summary: %v", reply)
```

Comment:
Don't use global logger.

Reply:
Global loggers are unfortunately used widely around client/uplink code. To pass common log instance I would need to go and update things like segments.Store, ecclient.

Reply:
Oh well, we'll fix that with other storage node fixes then.